website/docs/r/storage_account.html.markdown

```diff
@@ -229,6 +233,22 @@ A `queue_properties` block supports the following:
 
 * `hour_metrics` - (Optional) A `hour_metrics` block as defined below.
 
+---
+
+A `blob_account_properties` block supports the following:
+
+* `static_website` - (Optional) A `static_website` block as defined below.
+
+---
+
+A `static_website` block supports the following:
+
+* `enabled` - (Required) Indicates whether static website support is enabled for the given account.
+
+* `index_document` - (Optional) The webpage that Azure Storage serves for requests to the root of a website or any subfolder. For example, index.html. The value is case-sensitive.
```

Comment:
Should the documentation mention the default value for index and 404?

Reply:
I believe they both default to no value. From my testing, if no index document is specified, the primary web endpoint just returns a 404. If no 404 document is specified, Azure will return a default generic 404 page (added note for this).